package.json

```diff
@@ -28,13 +28,15 @@
     "babel-loader": "^6.2.0",
```

Comment:
you can drop the babel-loader and json-loader.  Looks like gulp-babel is a relic too?